Assets/Scripts/Models/Settings.cs

```diff
@@ -39,9 +39,18 @@ public static class Settings
     private static string userSettingsFilePath = System.IO.Path.Combine(
         Application.persistentDataPath, "Settings.xml");
 
+    private static bool unsavedSettings = false;
+
     static Settings()
     {
         LoadSettings();
+
+        PrototypeManager.SchedulerEvent.Add(
+            new Scheduler.ScheduledEvent(
+                "Settings_SaveSettings",
+                (evt) => Settings.SaveSettings()));
+
+        Scheduler.Scheduler.Current.ScheduleEvent("Settings_SaveSettings", 10f, true);
```

Comment:
I had originally thought that changing a setting would spawn a once off event with a delay of like .1 seconds or something -- just to bump the saving to the next frame. But this is probably fine too.